frontend/src/scenes/events/eventDefinitionsLogic.ts

```diff
@@ -18,6 +18,10 @@ interface EventsGroupedInterface {
 export const eventDefinitionsLogic = kea<
     eventDefinitionsLogicType<EventDefinitionStorage, EventDefinition, EventsGroupedInterface, SelectOption>
 >({
+    actions: () => ({
+        updateEventDefinition: (id: string, description: string | null) => ({ id, description }),
+        setEventDefinitions: (event) => ({ event }),
```

Comment:
Nit: is not clear what this action is intended for purely from the name

Reply:
This will definitely be updated since we'll be adding tags and stuff